state-chain/pallets/cf-validator/src/lib.rs

```diff
@@ -84,7 +85,18 @@ pub enum RotationPhase<T: Config> {
 	NewKeysActivated(RuntimeRotationState<T>),
 	SessionRotating(Vec<ValidatorIdOf<T>>, <T as Chainflip>::Amount),
 }
-
+impl<T: pallet::Config> Display for RotationPhase<T> {
```

Comment:
Better to make this std-gated and use core::fmt instead of importing via scale.

```suggestion
#[cfg(feature = std)]
impl<T: pallet::Config> core::fmt::Display for RotationPhase<T> {
	fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
	//...
	}
}
```